app/src/main/java/org/p2p/wallet/swap/ui/jupiter/main/widget/SwapWidgetModel.kt

```diff
@@ -43,13 +25,13 @@ sealed interface SwapWidgetModel {
     ) : SwapWidgetModel
 }
 
-private fun bigSkeleton(): SkeletonCellModel = SkeletonCellModel(
+fun bigSkeleton(): SkeletonCellModel = SkeletonCellModel(
```

Comment:
are you sure we need  to make bigSkeleton public? it's very ambiguous

Reply:
create SkeletonExtensions.kt